q2_feature_table/tests/filter/test_conditional_filter.py

```diff
@@ -40,6 +40,38 @@ def test_conditional(self):
         npt.assert_array_equal(known.ids(axis='observation'),
                                test_.ids(axis='observation'))
 
+    def test_allow_empty_table_true(self):
+        # test True
```

Comment:
```suggestion
```
Since this test method has a clearly defined test case that is reflected in the name, the comment isn't necessary here.